IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -175,7 +191,12 @@ - (void)setupCancelButton
 
 - (UIActivityIndicatorView *)prepareLoadingIndicator:(UIView *)rootView
 {
-    UIActivityIndicatorView *loadingIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleWhiteLarge];
+    UIActivityIndicatorView *loadingIndicator;
+#if TARGET_OS_UIKITFORMAC
```

Comment:
can we unify those two and use same spinner for both?
What's the reason to have those different?

Reply:
UIActivityIndicatorViewStyleWhiteLarge is deprecated in iOS 13.
UIActivityIndicatorViewStyleLarge is **only** availabe in iOS 13.

Reply:
ok, so should we check for iOS 13+ instead then?

Reply:
done